src/bosh-softlayer-cpi/action/create_vm_test.go

```diff
@@ -11,6 +11,7 @@ import (
 
 	. "bosh-softlayer-cpi/softlayer/common"
 	fakescommon "bosh-softlayer-cpi/softlayer/common/fakes"
```

Comment:
I've created a separate test for the case with 64 character long name, hope it makes it clearer. That's the only 'bad' case we could have here.